src/iosrv/init.c

```diff
@@ -161,7 +161,7 @@ register_dbtree_classes(void)
 }
 
 static int
-modules_load(uint64_t *facs)
+modules_load(const char *op, int (*func)(const char *, void *), uint64_t *facs)
```

Comment:
a little bit duplicate for these two parameters  op and func?

Reply:
Yeah, I don’t like it.   Let me rework it a bit